### ISSUE TITLE
[Feature]: About Us section in the home page

### ISSUE THREAD

**Issue by Srijita-Mandal** [aman34503/Go-Airbnb#36]
### Is your feature request related to a problem? Please describe.

There is no particular about us page or section in the home page. I want to add about us section in the home page with improved UI of the page.

Can I work this issue under JWOC?

**Comment by aman34503** [comment]
yes ! make sure page is Responsive and also Put some Relevent Content There @Srijita-Mandal

**Comment by Srijita-Mandal** [comment]
@aman34503  Can you please give me one more day.. I'll definitely solve the issue. I was sick from last two days actually.
